onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -223,7 +223,7 @@ def load_pretrained_model(model_name, config, cache_dir, custom_model_class, if_
     transformers_module = __import__("transformers", fromlist=[model_class_name])
     model_class = getattr(transformers_module, model_class_name)
 
-    return model_class.from_pretrained(model_name, config=config, cache_dir=cache_dir)
+    return model_class.from_pretrained(model_name, config=config, cache_dir=cache_dir, use_cdn=False)
```

Comment:
why set use_cdn to False for all models? Its spec says only model > 20g needs to set it to False.

Reply:
only t5-11b requires that actually. I think it's ok to set it to false for all models(may lower download speed according to spec). But It's better to only set for t5-11b. Will modify it.